### PR TITLE
adjust light theme to IDA 9.0

### PR DESCRIPTION
**Pull request by LittFlower**

### CODE REVIEW

File: Rx-VSCDesign-light/theme.css [hunk 1]
```diff
@@ -26,74 +26,84 @@ hexview_t {
...
+QToolButton {
+  min-width: 1.2em;
+  height: 1.2em;
+  border: 0px solid black;
   background: transparent;
-  border    : 1px solid #eeeeee;
 }
 
-QToolBar:handle {
-  image        : url("$RELPATH/icons/drag.png");
```
Comment (Reverier-Xu) [inline Rx-VSCDesign-light/theme.css:44]: Where is the handle?

**Review by Reverier-Xu — CHANGES_REQUESTED** [review]
Please test and fix the comment here.

File: Rx-VSCDesign-light/theme.css (hunk 1 above)
Comment (LittFlower) [reply Rx-VSCDesign-light/theme.css:44]: fixed.

**Comment by LittFlower** [comment]
I have testd.

File: Rx-VSCDesign-light/theme.css [hunk 2]
```diff
@@ -415,28 +426,28 @@ DockWidgetTitle[active="true"] {
...
 
 DockWidgetTitle QPushButton {
-  min-height      : 0.9em;
-  height          : 0.9em;
-  min-width       : 1.1em;
-  padding         : 0;
+  min-height: 0.9em;
+  height: 0.9em;
+  min-width: 1.1em;
+  padding: 0;
```
Comment (Reverier-Xu) [inline Rx-VSCDesign-light/theme.css]:
```suggestion
  min-height: 1.2em;
  height: 1.2em;
  min-width: 1.2em;
  padding: 0px;
```

File: Rx-VSCDesign-light/theme.css [hunk 3]
```diff
@@ -397,13 +408,13 @@ DockWidget {
 
 DockWidgetTitle {
   background: #cfcfcf;
-  height    : 1.2em;
+  height: 1.2em;
   max-height: 1.2em;
   min-height: 1.2em;
```
Comment (Reverier-Xu) [inline Rx-VSCDesign-light/theme.css]:
```suggestion

```

File: Rx-VSCDesign-light/theme.css [hunk 4]
```diff
@@ -397,13 +408,13 @@ DockWidget {
...
-  padding   : 0px;
-  border    : 0;
-  margin    : 0;
-  color     : #000000;
+  padding: 0px;
```
Comment (Reverier-Xu) [inline Rx-VSCDesign-light/theme.css]:
```suggestion
  padding: 0.1em 0.2em;
```

File: Rx-VSCDesign-light/theme.css [hunk 5]
```diff
@@ -291,16 +302,16 @@ QMenu:icon {
...
+  width: 1px;
+  margin: 6px;
 }
 
 QComboBox {
   background: #dfdfdf;
-  border    : 1px solid #afafaf;
-  padding   : 6px;
+  border: 1px solid #afafaf;
+  padding: 6px;
```
Comment (Reverier-Xu) [inline Rx-VSCDesign-light/theme.css]:
```suggestion
  padding: 0.15em 0.3em;
```

**Review by Reverier-Xu — CHANGES_REQUESTED** [review]
Please ensure that:

1. all basic components (buttons, combo box, line edit) have the same height;
2. tabs and dock title have the same height and padding;

File: Rx-VSCDesign-light/theme.css [hunk 6]
```diff
@@ -675,7 +735,7 @@ navigator_t {
 }
 
 QSplitter::handle {
-  image           : url(images/splitter.png);
+  image: url(images/splitter.png);
   background-color: white;
 }
```
Comment (Reverier-Xu) [inline Rx-VSCDesign-light/theme.css:741]:
```suggestion

QSplitter::handle:hover {
  background-color: #0078D6;
}
```

**Review by Reverier-Xu — CHANGES_REQUESTED** [review]
the handle should be highlighted when user hovers.

File: Rx-VSCDesign-light/theme.css [hunk 7]
```diff
@@ -26,74 +26,91 @@ hexview_t {
...
+QToolButton {
+  min-width: 1.2em;
+  height: 1.2em;
+  border: 0px solid black;
   background: transparent;
-  border    : 1px solid #eeeeee;
+}
+
+QToolButton:hover {
+  background: #20ffffff;
```
Comment (Reverier-Xu) [inline Rx-VSCDesign-light/theme.css]:
```suggestion
  background: #20000000;
```

button highlight in light theme should not be `#ffffff` or alpha values.

**Review by Reverier-Xu — CHANGES_REQUESTED** [review]

**Review by Reverier-Xu — APPROVED** [review]
thanks!

**Pull request merged**